en/docs/references/ballerina-expression-syntax.md

```diff
@@ -71,348 +71,348 @@ The boolean data type has one of the two possible values, `true` or `false`.
   false
   ```
 
-## Accessing Variables
+## Accessing variables
 
 ### Using the variable value
 
 Variables keep values in memory for later access. In an expression syntax-supported input field, you can refer to these variables directly using the variable name.
 
 #### Example
 
-- Converting the value of the integer variable (`total`) to a string value.
+- This converts the value of an integer variable (**total**) to a string:
   ```ballerina
   total.toString()
   ```
 
-## Supported Operators
+## Supported operators
 
-### String Concatenation
+### String concatenation
 
-You can use the `+` operator to do string concatenations. The string value can come from a string literal or a variable.
+You can use the **+** operator for string concatenations. The string value can come from a string literal or a variable.
 
 #### Examples
 
-- Concatenating two string literals.
+- Concatenates two string literals:
  
   ```ballerina
   "Hello " + "world!" 
   ```
   
-- Concatenating a string literal and a variable (`name`).
+- Concatenates a string literal and a variable (**name**):
```

Comment:
I don't agree to have bold for code variables and syntaxes. IMO, we should use backticks for them and use bold only for UI elements and texts.